addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -23,9 +23,72 @@ def _default_email_to(self):
             ('create_uid', '=', self.env.uid),
         ], order='create_date desc', limit=1).email_to or self.env.user.email_formatted
 
+    def _get_model_selection(self):
+        result = self.env["ir.model"].sudo().search([("transient", "=", False)])
+        return [(model.model, model.name) for model in result]
+
     email_to = fields.Text(string='Recipients', required=True,
                            help='Carriage-return-separated list of email addresses.', default=_default_email_to)
     mass_mailing_id = fields.Many2one('mailing.mailing', string='Mailing', required=True, ondelete='cascade')
+    email_from = fields.Char(string='From', related='mass_mailing_id.email_from')
+    subject = fields.Char(string='Subject', compute='_compute_subject')
+    body_html = fields.Html(string='Preview Body', compute='_compute_body_preview', sanitize='email_outgoing')
```

Comment:
Bloups @tde-banana-odoo 

Need a quick technical and security assist on this one.
This new field will be used to render a preview of the mass mailing on a specific record:
<img width="1009" height="1133" alt="image" src="https://github.com/user-attachments/assets/747df53a-2b1f-4544-9e0b-7e00a1cff76d" />

As this is effectively an "outgoing email", this is the correct choice of sanitize here?
Otherwise tags and attributes used by the mailing rendering process will be stripped?

Thanks for the help & cheers!

cc @abd-msyukyu-odoo